src/pages/Share/index.tsx

```diff
@@ -1,28 +1,43 @@
-import { IonButton, IonContent, IonPage, IonTitle, IonToolbar, IonItemSliding, IonItemOptions, IonItemOption, IonSpinner, IonList, IonItem, IonLabel, IonListHeader, IonAvatar, IonIcon, IonButtons, IonModal, IonImg, IonHeader, IonProgressBar, IonText } from '@ionic/react'
-import { share, trash, addCircleOutline } from 'ionicons/icons'
-import { useState } from 'react'
-import { DateTime } from 'luxon'
-import createPersistedState from 'use-persisted-state'
-import { getThumbnailUrl } from 'image-thumbnail-generator'
-import { transform, transformForShare, useNodes } from '../../util/ipfs'
-import upload, { maxChunkSize } from '../../util/upload'
-import PageContainer from '../../components/PageContainer'
-import FileIcon from '../../components/FileIcon'
-import './index.scss'
+import {
+  IonButton,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonList,
+  IonItem,
+  IonLabel,
+  IonModal,
+  IonImg,
+  IonHeader,
+  IonProgressBar,
+  IonText,
+  IonThumbnail,
+} from "@ionic/react"
+import { useState } from "react"
+import { DateTime } from "luxon"
+import createPersistedState from "use-persisted-state"
+import { getThumbnailUrl } from "image-thumbnail-generator"
+import { transformForShare } from "../../util/ipfs"
+import upload, { maxChunkSize } from "../../util/upload"
+import PageContainer from "../../components/PageContainer"
+import FileIcon from "../../components/FileIcon"
+import QRCode from 'react-qr-code'
+import "./index.scss"
 
 const BIG_FILE_THRESHOLD = 5 * 1024 // 5mb
 const PROGRESS_THRESHOLD = maxChunkSize / 1000
 
 type Upload = {
   name: string
+  cid: string
   url: string
   mimeType: string
   extension?: string
   thumbnail?: string
   date: string
 }
-const useUploadedFiles = createPersistedState<Upload[]>('uploaded-files')
-const hasNativeShare = typeof navigator.share === 'function'
+const useUploadedFiles = createPersistedState<Upload[]>("uploaded-files")
+const hasNativeShare = typeof navigator.share === "function"
```

Comment:
A lot of these changes in the PR are quote style changes - if you run the `lint` command it should conform everything to our standard and remove a lot of these diffs.